### DISCUSSION TITLE
Set up layouts faster - Tkinter Quick Layout

### DISCUSSION THREAD

**Discussion by EasyDevv** [TomSchimansky/CustomTkinter#2028] [Showcase]
[Tkinter Quick Layout](https://github.com/EasyDevv/Tkinter_Quick_Layout)
---

Hi, I was having a hard time creating a complex layout with just code

I created this tool because I wanted to be able to create layouts while previewing them.

It actually speeds up my work.

Enjoy 🙂

[Go to my repository](https://github.com/EasyDevv/Tkinter_Quick_Layout)

Basic Control
---
▲ ▼ : Row
◀ ▶ : Column
▲▲ ▼▼ : Row span
◀◀ ▶▶ : Column span
✓ : Copy to clipboard
X : Remove Frame

Image
---
![tkinter_quick_layout](https://github.com/TomSchimansky/CustomTkinter/assets/11148818/35887086-a3fd-48c6-807f-6a41049fafb8)

**Comment by Akascape** [comment]
@EasyDevv where is your repository?

**Reply by EasyDevv** [reply]
I fixed the repository not showing up. thank you
